packages/constants/src/index.js

```diff
@@ -212,6 +212,43 @@ export const TYPES = {
     UNSIGNED_SHORT_5_5_5_1: 32820,
     FLOAT: 5126,
     HALF_FLOAT: 36193,
+
+    /**
+     * Returns if any type in the `TYPES` enum is of integer
+     * nature.
+     *
+     * @param {number} glint - `TYPES` types
+     * @return if `glint` is an integral type
+     */
+    isInteger(glint)
```

Comment:
Theses are probably better in the utils package

Reply:
@bigtimebuddy Hey, I've solved that problem with #9282 (commit in `batch-dev`). I don't need `TYPES.isInteger`; hence, I've eliminated that.

In addition, I have moved `TYPES.sizeOf` to `@pixi/utils/src/data/sizeOfType.js`. It has been renamed to `sizeOfType`.